testsuite/tools.py

```diff
@@ -15,10 +15,10 @@
 
 _tr = {
     "echo_api": "echo-api+ssl",
-    "httpbin": "httpbin+ssl",
-    "httpbin_nossl": "httpbin",
+    "httpbin": "mockserver+ssl",
```

Comment:
This just adds more confusion and removes any remaining transparency the tools had.